django/contrib/admin/options.py

```diff
@@ -1704,8 +1721,8 @@ def _delete_view(self, request, object_id, extra_context):
 
         # Populate deleted_objects, a data structure of all related objects that
         # will also be deleted.
-        (deleted_objects, model_count, perms_needed, protected) = get_deleted_objects(
-            [obj], opts, request.user, self.admin_site, using)
+        (deleted_objects, model_count, perms_needed, protected) = self.get_deleted_objects(
```

Comment:
I think that you can remove brackets
`deleted_objects, model_count, perms_needed, protected = ...`